app.py

```diff
@@ -1,21 +1,57 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import requests
+import json
+import Trello
+import os
 
 app = Flask(__name__)
-app.config.from_object('flask_config.Config')
 
-@app.route('/', methods = ['POST', 'GET', 'PUT', 'DELETE'])
+@app.route('/', methods=['GET', 'POST', 'PUT'])
 def index():
-    if request.method == 'POST':
-        NewItem = request.form["NewItem"]
-        session.add_item(NewItem)
-        return render_template('index.html', items = session.get_items())
-    elif request.method == 'PUT':
-        SelectedItem = requestform["CompleteTask"]
-        session.save_item()
-        return render_template('index.html', items = session.get_items())
-    else:
-        return render_template('index.html', items = session.get_items())
+    return render_template(
+        'index.html', 
+        lists = Trello.lists, 
+        boards = Trello.boards,
+        cards_from_things_to_do = Trello.cards_from_things_to_do(), 
+        cards_from_doing = Trello.cards_from_doing(), 
+        cards_from_done = Trello.cards_from_done()
+    )
+
+@app.route('/add_thing_to_do/', methods=['POST'])
+def add_thing_to_do(): 
+    NewThingToDo = request.form['NewThingToDo']
+    requests.post(Trello.post_things_to_do, params={**Trello.trello_params, 'name': NewThingToDo})
```

Comment:
These sections might be nicer as functions inside the Trello file, saving us from exposing quite as many details (like trello_params, the relevant URLs etc)